### DISCUSSION TITLE
Welcome to h5p-bundle Discussions!

### DISCUSSION THREAD

**Discussion by jorisdugue** [jorisdugue/h5p-bundle#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iflorespaz** [comment]
Did you saw H5P plans to make a OER Hub, so users all over the world can share their H5P content 👌
https://h5p.org/node/778895
This will make a huge difference for teachers in classroom or in everyday life .... it is hard to sit alone with all issues !
what do you think?

**Reply by jorisdugue** [reply]
> Did you saw H5P plans to make a OER Hub, so users all over the world can share their H5P content 👌
> https://h5p.org/node/778895
> This will make a huge difference for teachers in classroom or in everyday life .... it is hard to sit alone with all issues !
> what do you think?
I think this is a really good idea to see if a lot of people want to share their own content with others 😄
